### DISCUSSION TITLE
Updated Docker with the pre-release

### DISCUSSION THREAD

**Discussion by ryansheehan** [Pryaxis/TShock#1788] [Show and tell]
I've update the docker image with the v1.4.0.2 pre-release.  For those that want an easy button to run a server on linux, checkout docker!  I typed up a bunch of instructions on [docker hub](https://hub.docker.com/r/ryshe/terraria).

You can find the source [here](https://github.com/ryansheehan/terraria).

**Comment by QuiCM** [comment]
That's really cool! Thank you!

**Comment by hakusaro** [comment]
Wow, that's sweet of you! Thank you!
